plugins/gatsby-source-jenkinsplugins/utils.js

```diff
@@ -296,9 +312,13 @@ const fetchPluginVersions = async ({createNode, reporter}) => {
             url	"https://updates.jenkins.io/download/plugins/42crunch-security-audit/2.1/42crunch-security-audit.hpi"
             version 2.1
             */
+            const date = dateFNs.parse(data.buildDate, 'MMMM d, yyyy', new Date(0));
+            if (!firstReleases[data.name] || firstReleases[data.name].getTime() > date.getTime()) {
```

Comment:
is this condition right?

Won't the right hand side never work because the it's not contained in firstReleases?

Reply:
what this statements tries to do is to store a release in the `firstReleases` map if
* either no release was stored
* or the previously stored release is newer than the currently parsed one

Judging by the "new plugins" on the front page it works. Did I miss something?

Reply:
ah I guess it's if doesn't exist or time later

Reply:
@timja OK to merge or is there still something unclear?